shell/pom.xml

```diff
@@ -69,5 +69,45 @@
       <artifactId>alluxio-core-common</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
```

Comment:
Do you think we can load the UFS jars in the classpath when running the shell commands, but not put them in the assembly-client.jar?